lib/bulk/common.js

```diff
@@ -242,25 +251,30 @@ class BulkWriteResult {
 }
 
 /**
- * Create a new WriteConcernError instance (INTERNAL TYPE, do not instantiate directly)
- *
- * @class
- * @return {WriteConcernError} a WriteConcernError instance
+ * @classdesc An error representing a failure by the server to apply the requested write concern to the bulk operation.
  */
 class WriteConcernError {
+  /**
+   * Create a new WriteConcernError instance (INTERNAL TYPE, do not instantiate directly)
```

Comment:
nit: we have traditionally called out notes in bold, like:
```
Create a new WriteConcernError

**NOTE:** Internal type, do not instantiate directly
```
I don't know if there's another jsdoc tag for this